datastore/google/cloud/datastore/client.py

```diff
@@ -648,3 +648,33 @@ def do_something(entity):
         if "namespace" not in kwargs:
             kwargs["namespace"] = self.namespace
         return Query(self, **kwargs)
+
+    def reserve_ids(self, complete_key, num_ids):
+        """Prevents the supplied keys' IDs from being auto-allocated by Cloud
```

Comment:
Please use "imperative voice" for the docstring summary, and keep it to one line, e.g.:

```python
    def reserve_ids(self, complete_key, num_ids):
        """Prevent the supplied keys' IDs from being auto-allocated.

        ...
```